modules/cpdos/hmo.py

```diff
@@ -6,38 +6,85 @@
 https://cpdos.org/#HMO
 """
 
-from ..utils import * 
+from modules.utils import requests, random, logging
+
+logger = logging.getLogger(__name__)
 
 VULN_NAME = "HTTP Method Override"
 
-def HMO(url, s, main_len, main_status_code, authent):
-    #HTTP Method Overcide
+CONTENT_DELTA_RANGE = 1000
+
+
+def HMO(url, s, initial_response, authent):
+    """Function to test for HTTP Method Override vulnerabilities"""
+
     methods = ["POST", "PUT", "HELP", "DELETE"]
```

Comment:
Only these verbs at the moment ?

Potentially adding ?: "GET", "HEAD", "OPTIONS", "CONNECT", "PATCH", "TRACE", "NONSENSE"

Reply:
```suggestion
    methods = [
        "GET"
        "POST",
        "PUT",
        "HELP",
        "DELETE",
        "HEAD",
        "OPTIONS",
        "CONNECT",
        "PATCH",
        "TRACE",
        "NONSENSE",
    ]
```